docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>I like this example even if it is with a for loop it works well enough I think :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>